### ISSUE TITLE
Cannot find module './GeosuggestItem.jsx'

### ISSUE THREAD

**Issue by e-monson** [ubilabs/react-geosuggest#16]
When I build my app with gulp and browserify, I get the error: 
>Cannot find module './GeosuggestItem.jsx' from /blahblah/node_modules/react-geosuggest/module 

When I look at the directory specified in the error, there is a file "GeosuggestItem.js" (without the "x"). Why would it be looking for GeosuggestItem.jsx instead?

**Comment by ro-ka** [comment]
That is strange as in the package.json, the main file is correctly specified: https://github.com/ubilabs/react-geosuggest/blob/master/package.json#L5
Any ideas?

**Comment by e-monson** [comment]
I think it may be hung up here:
https://github.com/ubilabs/react-geosuggest/blob/master/src/Geosuggest.jsx#L4

**Comment by kkarimi** [comment]
Yep changing the line to .js fixes this issue

**Comment by lPadier** [comment]
The simplest solution is to change line 4 of Geosuggest.jsx from :
```javascript
 GeosuggestItem = require('./GeosuggestItem.jsx');
```
to 
```javascript
 GeosuggestItem = require('./GeosuggestItem');
```
as Babel will look for the file using .js, .es6 and .jsx patterns, and node will look for the file using .js extension

**Comment by ro-ka** [comment]
Thanks for the PR, released as `1.4.2`!
